pkg/context/keys.go

```diff
@@ -1,11 +1,17 @@
 package context
 
 const (
-	// UsernameKey is the context key for preferred_username claim
+	// UsernameKey is the context key for the preferred_username claim
 	UsernameKey = "username"
-	// EmailKey is the context key for email claim
+	// EmailKey is the context key for the email claim
 	EmailKey = "email"
-	// SubKey is the context key for subject claim
+	// NameKey is the context key for the given name claim
+	GivenName = "givenName"
+	// FamilyNameKey is the context key for the family name claim
+	FamilyNameKey = "familyName"
+	// Company is the context key for the company claim
+	Company = "company"
```

Comment:
```suggestion
	// GivenNameKey is the context key for the given name claim
	GivenNameKey = "givenName"
	// FamilyNameKey is the context key for the family name claim
	FamilyNameKey = "familyName"
	// CompanyKey is the context key for the company claim
	CompanyKey = "company"
```

Reply:
fixed as part of: 809fe7555398f3cfc60debeeb9f8c8cab16ec423